salt/orchestrate/aws/cloud_profiles/ocw-production-aka.conf

```diff
@@ -0,0 +1,24 @@
+# -*- mode: yaml; coding: utf-8; -*-
+ocw-production-aka:
+  provider: mitx
+  size: t3.small
+  image: {{ salt.sdb.get('sdb://consul/debian_ami_id')|default('ami-03006931f694ea7eb') }}
+  ssh_username: admin
+  ssh_interface: private_ips
+  script_args: -Z -A salt.private.odl.mit.edu
+  iam_profile: ocw-production-instance-role
+  ebs_optimized: True
+  tag:
+    role: ocw-aka
```

Comment:
We should probably update the naming to be more obvious since `aka` is arbitrary. Possibly something like `origin-server`